File: justorder_framework/src/main/java/it/polito/justorder_framework/model/Product.java
package it.polito.justorder_framework.model;

import com.google.firebase.database.DataSnapshot;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Product {
    private String key;
    private String name;
    private Double cost;
    private String notes;
    private List<String> ingredients;
    private String category;
}

File: justorder_framework/src/main/java/it/polito/justorder_framework/model/Product.kt
package it.polito.justorder_framework.model

class Product: ModelWithVote() {
    var name: String = ""
    var imageUri: String? = null
    var cost: Double = 0.0
    var notes: String = ""
    var ingredients: List<String> = mutableListOf()
    var category: String = ""
}